packages/linked-data-proof/src/ProofSet.ts

```diff
@@ -187,7 +187,7 @@ export class ProofSet {
       purpose,
       documentLoader,
       expansionMap,
-      compactProof = true // consider not supporting anything else...
+      compactProof = false // digital bazaar doesn't have this option
```

Comment:
> BREAKING: Remove compactProof parameter when signing and verifying. This means that going forward, documents are required to use the appropriate contexts for the proof that they're using (error will be thrown otherwise).

https://github.com/digitalbazaar/jsonld-signatures/blob/9a665c5b712ca997b9ca8205de11fb6f6ae15fe0/CHANGELOG.md

Reply:
Ok, I'll take it out entirely in that case on our side 👍

Reply:
Do not do that, document the behavior fully, and preserve the defaults and configurability.